### PR TITLE
Add Azure CV Connector Implementation

### PR DESCRIPTION
**Pull request by lafernando**
$subject.

### CODE REVIEW

**Comment by daneshk** [comment]
@lafernando Since the module name is `azure.cv`, the directory structure should be `src/azure.cv`. Shall we change the directory name `azurecv` to `azure.cv`

File: src/azurecv/azurecv_connector.bal [hunk 1]
```diff
@@ -0,0 +1,85 @@
+// Copyright (c) 2018 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (daneshk) [inline src/azurecv/azurecv_connector.bal:1]:
```suggestion
// Copyright (c) 2020 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

File: src/azurecv/azurecv_constants.bal [hunk 2]
```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2018 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (daneshk) [inline src/azurecv/azurecv_constants.bal:1]:
```suggestion
// Copyright (c) 2020 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

File: src/azurecv/azurecv_types.bal [hunk 3]
```diff
@@ -0,0 +1,26 @@
+//
+// Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (daneshk) [inline src/azurecv/azurecv_types.bal:2]:
```suggestion
// Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

File: src/azurecv/tests/test.bal [hunk 4]
```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2018 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (daneshk) [inline src/azurecv/tests/test.bal:1]:
```suggestion
// Copyright (c) 2020 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

**Comment by lafernando** [comment]
> @lafernando Since the module name is `azure.cv`, the directory structure should be `src/azure.cv`. Shall we change the directory name `azurecv` to `azure.cv`

@daneshk Yes, missed that earlier, have changed it now, thanks!

**Pull request merged**